Guide.docc/CommonProblems.md

```diff
@@ -108,36 +108,35 @@ Global _reference_ types present an additional challenge, because they
 are typically not `Sendable`.
 
 ```swift
-class Chicken {
-    let name: String
-    var currentHunger: HungerLevel
+class WindowStyler {
+    var background: ColorComponents
 
-    static let prizedHen = Chicken()
+    static let defaultStyler = WindowStyler()
 }
 ```
 
 The problem with this `static let` declaration is not related to the
 mutability of the variable.
-The issue is `Chicken` is a non-Sendable type, making its internal state
+The issue is `WindowStyler` is a non-`Sendable` type, making its internal state
```

Comment:
This is definitely an improvement over "abstract" wording.

I'm tempted to push this a bit further though, given our "not only apps" messaging about swift I'm concerned about these examples invoking very "app centric" vibe? How about we pick some random "business logic" ideas, like some "decision maker" or some storage etc...?

WDYT @hborla?

Reply:
Ok here's another take on it: these are okey, but let's mix up examples a bit -- right now a lot of them are app-ish, so when we do a another pass of reviewing let's make sure there's a healthy mix of "app centric" and "not so app centric" examples. 

The examples can really be the same, it's just about their framing and type names tbh. We can do this as one of the last passes maybe

Reply:
Ok, I'd like to get this particular change in for now just to keep things consistent. But I definitely do think I over-corrected. Plus honestly I put very little thought into these names, so I think improvements here will be easy.